src/asmcnc/skavaUI/popup_info.py

```diff
@@ -746,7 +746,7 @@ def __init__(self, screen_manager, localization, description = False):
                       content=layout_plan,
                       size_hint=(None, None),
                       size=(500, 200),
-                      auto_dismiss= False
+                      auto_dismiss= True

```

Comment:
Why did you make this change?

Reply:
Sometimes the wait popups can get stuck (if something goes wrong - there used to be one with the software update that wouldn't clear if the SW was already on the latest version I think). I wanted to put an "X" in the corner but I couldn't find a straightforward way to do that (maybe a float layout would do it but I'm not sure). I did however find this auto-dismiss parameter that will clear the popup if the user presses on space that isn't the popup. This won't clear popups that are locked on by active processes that cause the UX to lock up (e.g. mid SW updates or downloading logs), but it is useful for cases where something has prevented the wait popup from clearing. 

There was a ticket in the backlog for this anyway (which I've since updated) and it was a quick change I could make while I was putting in the wait popup anyway :)

Reply:
Seems logical, I only highlighted this because I can imagine a user clearing a wait popup that shouldn't be cleared before its time has run out which could mess with functionality, such as the wait popup before the nudge screen that I've used in job recovery. In theory a user wouldn't do this so it might be fine but it just didn't feel completely robust. What do you think?

Reply:
On the other hand this may help the user avoid getting locked into a wait popup in job recovery in case some weird bug happens with the machine state... might be fine actually

Reply:
I understand, I had the same thought. I personally think that users are unlikely to click out of it most of the time, and it mainly acts as a "safety" if there are issues for any reason.  I think this way is the lesser of two evils. 

That said, if you think that it would be better to not have the auto-dismiss, then I can change it back and we'll keep everything as is :)

Reply:
I think it's probably fine :)